src/AzStorage.jl

```diff
@@ -284,6 +284,62 @@ x = read!(io, zeros(10))
 """
 Base.write(o::AzObject, data) = write(o.container, o.name, data)
 
+"""
+    writedlm(container, "blobname")
+
+Write the array `data` to a delimited blob with the name `blobname` in container `container::AzContainer`
+"""
+function DelimitedFiles.writedlm(c::AzContainer, o::AbstractString, data::AbstractArray)
+    tfile = tempname()
```

Comment:
Instead of making a temp file, we could just use an `IOBuffer`.  So something like:
```julia
io = IOBuffer(;write=true)
writedlm(io, data)
write(c, o, String(take!(io)))
```

Reply:
changed to using IOBuffer